src/utils/ETools/audio.cpp

```diff
@@ -366,11 +375,15 @@ int wav_open(FILE* in, oe_enc_opt* opt, unsigned char* oldbuf, int buflen)
      */
 
     if (!find_wav_chunk(in, "fmt ", &len))
+    {
+        free(wav);
```

Comment:
We can just move the `wav` allocation down to the place where it's actually used.